AccessCheckoutSDK/AccessCheckoutSDKTests/validation/model/CardValidationStateHandlerTests.swift

```diff
@@ -273,16 +310,7 @@ class CardValidationStateHandlerTests: XCTestCase {
         verify(merchantDelegate).expiryDateValidChanged(isValid: true)
     }
 
-    func testNotifyMerchantOfExpiryDateValidationState_notifiesMerchantOfInvalidExpiryDate() {
-        let validationStateHandler = CardValidationStateHandler(
-            merchantDelegate: merchantDelegate, expiryDateValidationState: false)
-
-        validationStateHandler.notifyMerchantOfExpiryDateValidationState()
-
-        verify(merchantDelegate).expiryDateValidChanged(isValid: false)
-    }
-
-    // MARK: CvcValidationStateHandler
+    // MARK: CvcValidationStateHandler (unchanged)
```

Comment:
review - we are missing
- `testHandleCvcValidation_shouldNotNotifyMerchantDelegateWhenCvcValidationStateDoesNotChangeFromTrue`
- `testHandleCvcValidation_shouldNotifyMerchantDelegateWhenCvcValidationStateChangesFromTrue`
- `testNotifyMerchantOfCvcValidationState_notifiesMerchantOfInvalidCvc`